### PR TITLE
[ZEPPELIN-585] Add new doc on how to embed Note\Paragraph plots into …

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Sorry for the late PR.

According to the user feedback, seems that it's not clear how to **embed output plots into other websites**.
So I made a documentation for explaining this feature.

### What type of PR is it?
Documentation

### Todos
* [x] - Write a documentation for How can we publish Zeppelin paragraph results

### Is there a relevant Jira issue?
[ZEPPELIN-585](https://issues.apache.org/jira/browse/ZEPPELIN-585)

### How should this be tested?
After apply this PR, just click `Quick Start -> Publish your Paragraph

### Screenshots (if appropriate)
picture
picture2

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/manual/publish.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Zeppelin provides a feature for publishing your notebook paragraph results. Using this feature, you can show Zeppelin notebook paragraph results in your own website. 
+It's very straightforward. Just use `<iframe>` tag in your page.
+
+### Copy a Paragraph Link
+A first step to publish your paragraph result is **Copy a Paragraph Link**.
+
+  * After running a paragraph in your Zeppelin notebook, click a gear button located on the right side. Then, click **Link this Paragraph** tab like below image.
+<center><img src="../assets/themes/zeppelin/img/docs-img/link-the-paragraph.png" height="100%" width="100%"></center>
+  
+  * Just copy the provided link. After then, change the host name to your ip address.
```
Comment (felixcheung) [inline docs/manual/publish.md]: I think changing host name only necessary if someone is running Zeppelin locally (like a laptop) but want it to be accessible outside of it? In normal use cases, Zeppelin should be running on a server?

Comment (AhyoungRyu) [inline docs/manual/publish.md]: @felixcheung You're right. I think I have to change this line for the normal use case : )

**Comment by AhyoungRyu** [comment]
@felixcheung Thank you for the review. I changed some sentences as your comments.

File: docs/manual/publish.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+{% include JB/setup %}
+
+## How can you publish your paragraph ?
+Zeppelin provides a feature for publishing your notebook paragraph results. Using this feature, you can show Zeppelin notebook paragraph results in your own website. 
+It's very straightforward. Just use `<iframe>` tag in your page.
+
+### Copy a Paragraph Link
+A first step to publish your paragraph result is **Copy a Paragraph Link**.
+
+  * After running a paragraph in your Zeppelin notebook, click a gear button located on the right side. Then, click **Link this Paragraph** tab like below image.
```
Comment (felixcheung) [inline docs/manual/publish.md]: I'm not sure what's the right name for it, maybe it's not a 'tab'? Is it a menu item?

Comment (AhyoungRyu) [inline docs/manual/publish.md]: @felixcheung I applied it too : )

**Comment by felixcheung** [comment]
looks good.

File: docs/index.md [hunk 3]
```diff
@@ -114,6 +111,8 @@ This way, you can easily embed it as an iframe inside of your website.</p>
   <img class="img-responsive center-block" src="/assets/themes/zeppelin/img/screenshots/publish.png" />
 </div>
 
+If you want to learn more about this feature, please checkout [here](./manual/publish.html).
```
Comment (corneadoug) [inline docs/index.md]: please checkout [here]
--> please visit [this page]

**Comment by corneadoug** [comment]
We might want to add some kind of disclaimer because of security issues with that feature:
* To embed the paragraph in a website, Zeppelin needs to reachable for that Website.
* With that iframe url, anybody that visit the website might be able to access Zeppelin if the allow-access is not properly set
* I'm not sure that this feature works if Shiro is activated on Zeppelin

**Comment by AhyoungRyu** [comment]
@corneadoug 
I applied your first issue in documentation. 
However, so far, I'm not sure how we can deal with your second issue. I just addressed as the current status of Zeppelin in this documentation. 
And I think the third one needs to be tested. So I'll test it and ping this PR again : )

**Comment by felixcheung** [comment]
"Zeppelin needs to `be` reachable for that Website" is probably better

**Comment by AhyoungRyu** [comment]
@felixcheung Shamed. You're right. A verb is needed after `to`. I pushed it again : )

**Comment by AhyoungRyu** [comment]
@corneadoug Sorry for the late response. I tested this feature with **Shiro on**. If Shiro is activated, the user need to enter **userid & password**  in their website like below image.
captureimg

Then, after loggin in, the published paragraph is shown. 
screen shot 2016-02-11 at 6 09 19 pm

Therefore, if anyone want to show their Zeppelin paragraph in their outside website, they need to deactivate Shiro authentication. (Actually, [Shiro is deactivated](https://github.com/apache/incubator-zeppelin/blob/master/conf/zeppelin-site.xml.template#L190) by default until the user activate it on purpose.)
Do I need to address this issue in the docs?

File: docs/manual/publish.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+For example,
+
+```
+<iframe src="http://< ip-address >:< port >/#/notebook/2B3QSZTKR/paragraph/...?asIframe" height="" width="" ></iframe>
+```
+
+Finally, you can see your beautiful visualization results in your web site. 
+<center><img src="../assets/themes/zeppelin/img/docs-img/your-website.png" height="90%" width="90%"></center>
+
+> **Note**: To embed the paragraph in a website, Zeppelin needs to be reachable for that Website.
```
Comment (corneadoug) [inline docs/manual/publish.md]: reachable for -> reachable by

File: docs/manual/publish.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+## How can you publish your paragraph ?
+Zeppelin provides a feature for publishing your notebook paragraph results. Using this feature, you can show Zeppelin notebook paragraph results in your own website. 
+It's very straightforward. Just use `<iframe>` tag in your page.
+
```
Comment (corneadoug) [inline docs/manual/publish.md]: We could have another Note under that line as a warning that says: 
> **Warning**: Use that feature with caution and in a trusted environment only, as Zeppelin entire Webapp could be accessible  for whoever visit that website

**Comment by AhyoungRyu** [comment]
@corneadoug Thanks! I added a warning sentence to this docs as you said.

**Comment by corneadoug** [comment]
LGTM

**Comment by felixcheung** [comment]
merging then...

**Pull request closed without merging**